packages/twenty-front/src/modules/ui/display/chip/components/EntityChip.tsx

```diff
@@ -38,10 +40,14 @@ export const EntityChip = ({
   maxWidth,
 }: EntityChipProps) => {
   const navigate = useNavigate();
-
   const theme = useTheme();
+  const { isReadOnly } = useContext(RecordTableRowContext);
```

Comment:
This one will exists only in RecordTable. Not ideal but this is what we want.

Reply:
this is coupling the EntityChip (from UI) t the recordTable which is not possible. Can we either pass it as Props when calling the component? I feel it should be actually part of the FieldContext (set by the RecordTable) and then the EntityChip gets it as props (cannot read directly from FieldContext either)

Reply:
also, what if we set linkToEntity to null? doesn't it work instead of adding a new behavior?

Reply:
Good point. I moved the logic to RecordChip to avoid the dependency with twenty-ui. And I send a link empty instead.
Not using FieldContext because a lot of places would require an update while this logic should be removed when we have the relations